async_rediscache/types/base.py

```diff
@@ -237,7 +127,7 @@ def _to_typestring(key_or_value: RedisKeyOrValue, prefixes: _PrefixTuple) -> str
 
     @staticmethod
     def _from_typestring(
-            key_or_value: Union[bytes, str], prefixes: _PrefixTuple
+        key_or_value: Union[bytes, str], prefixes: _PrefixTuple
```

Comment:
What's the point of this change? This is how PEP 8 advices hanging indents since the line of code is at the same level.

I believe I saw a few other changes like this one.

Reply:
I think I only made one other change like that. Either way, I did this way to confirm with the majority of the project which does this.

On a more tangential note, pep claims that the extra indent is for clearly defining the border between arguments and function code, but that's not really valid when the closing parenthesis are indented the way they are.